tally_ho/libs/verify/quarantine_checks.py

```diff
@@ -17,8 +17,19 @@ def create_quarantine_checks():
 
 def quarantine_checks():
     """Return tuples of (QuarantineCheck, validation_function)."""
-    methods = [pass_overvote, pass_tampering]
-    checks = QuarantineCheck.objects.all().order_by('pk')
+    all_methods = {'pass_overvote': pass_overvote,
+                   'pass_tampering': pass_tampering}
+    methods = []
+
+    quarantine_checks_methods =\
+        QuarantineCheck.objects.filter(
```

Comment:
do we want to order this?

Reply:
perhaps alphabetically

Reply:
Nice catch, forgot that and i will order by the primary key as we were doing initially when querying all checks. https://github.com/onaio/tally-ho/blob/cc0bff7b63825e828425327a28f872af3e3fa4f1/tally_ho/libs/verify/quarantine_checks.py#L21

Reply:
Done